pattern-1/conf/is/conf/datasources/bps-datasources.xml

```diff
@@ -31,7 +31,7 @@
             </jndiConfig>
             <definition type="RDBMS">
                 <configuration>
-                    <url>jdbc:mysql://wso2is-pattern1-mysql-service:3306/WSO2_IDENTITY_DB?autoReconnect=true&amp;verifyServerCertificate=false&amp;useSSL=true</url>
+                    <url>jdbc:mysql://wso2is-pattern1-mysql-service:3306/WSO2_BPS_DB?autoReconnect=true&amp;verifyServerCertificate=false&amp;useSSL=true</url>
```

Comment:
we will have to make useSSL to false, otheriwse newer connectors will fail in an out-of-the-box setup ?